conf/test.config

```diff
@@ -0,0 +1,24 @@
+/*
+========================================================================================
+    Nextflow config file for running minimal tests
+========================================================================================
+    Defines input files and everything required to run a fast and simple pipeline test.
+
+    Use as follows:
+        nextflow run nf-core/rts -profile test,<docker/singularity>
+
+----------------------------------------------------------------------------------------
+*/
+
+params {
+    config_profile_name        = 'Test profile'
+    config_profile_description = 'Minimal test dataset to check pipeline function'
+
+    // Limit resources so that this can run on GitHub Actions
+    max_cpus   = 2
+    max_memory = 6.8.GB
+    max_time   = 6.h
+
+    // Input data
+    input  = 'https://github.com/Waseju/rts-pipeline/blob/d7a3a2d363190e3d490db0472ff6b97ca82bb7fd/testdata/testdata.tar.gz?raw=true'
```

Comment:
Please include the test data also in this repository or in a repository part of the `qbic-pipelines` organization

Reply:
I think the test data is now under /assests/